platforms-serverless/azure-functions-linux/finally.sh

```diff
@@ -27,4 +26,7 @@ else
     # move function code back to placeholder location, so we can try again
     mv "$app" "func-placeholder"
 
+    # stop the function
+    # Note that re-running the test will create a new function from scratch
+    az functionapp stop --name "$app" --resource-group "$group"
```

Comment:
What does "stopping" a function really do?

Reply:
When the function is running, it's listening for HTTP requests, when it's stopped, it's not listening.

Reply:
Ah, get it. Suggestion: https://github.com/prisma/ecosystem-tests/pull/4440/files#r1446076654